src/components/TodoList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import style from './styles/TodoList.css';
+
+const TodoList = (props) => {
+    let tasks = props.items.map((item) => <li key={item.id} className={style.TodoListItem}>{item.text}</li>)
```

Comment:
brakuje jeszcze, zeby po kliknieciu w dany item, odpalana byla metoda, ktora usuwa element z listy to do